scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -157,3 +157,29 @@ SOP-18_7.0x12.5mm_P1.27mm:
   pitch: 1.27
   num_pins_x: 0
   num_pins_y: 9
+  
+STC_SOP-16_3.9x9.9mm_P1.27mm:
+  size_source: 'https://www.stcmicro.com/datasheet/STC15F2K60S2-en.pdf'
```

Comment:
Include the pointer to the correct page for such a large datasheet:

https://www.stcmicro.com/datasheet/STC15F2K60S2-en.pdf#page=154

Reply:
Done